priv/worker.py

```diff
@@ -140,7 +156,7 @@ def env_close(self, instance_id):
       self._remove_env(instance_id)
 
   def env_close_all(self):
-    for key in self.envs.keys():
+    for key in list(self.envs.keys()):
```

Comment:
It is not needed as now I convert the keys iterator into a list so I have a seperate list of all keys in the dict, even if we remove the environment from the dict, it will not cause conflict as the list of keys stays same throughout the job.
This is default behaviour of dict.keys() in python 2.7
python 3 sucks....